lightly/api/api_workflow_download_dataset.py

```diff
@@ -150,9 +148,38 @@ def lambda_(i):
             msg += 'Failed at image: {}'.format(results.index(False))
             warnings.warn(msg)
 
+    def download_embeddings_csv(self, output_path: str) -> None:
+        """Downloads the last created embedding from the dataset to the output path.
```

Comment:
The wording is a bit complicated, can we do "Downloads the latest embedding from the dataset and saves it to the output path"